app/components/UI/WalletAction/WalletAction.types.ts

```diff
@@ -1,14 +1,66 @@
 import { ImageStyle, ViewStyle } from 'react-native';
 import { IconName } from '../../../component-library/components/Icons/Icon';
 import { AvatarSize } from '../../../component-library/components/Avatars/Avatar';
+import { strings } from '../../../../locales/i18n';
+
+export enum WalletActionType {
+  Buy = 'Buy',
+  Sell = 'Sell',
+  Swap = 'Swap',
+  Bridge = 'Bridge',
+  Send = 'Send',
+  Receive = 'Receive',
+}
+
+export interface WalletActionStrings {
+  title: string;
+  description: string;
+  disabledDescription: string;
+}
+
+export const walletActionStrings: Record<
+  WalletActionType,
+  WalletActionStrings
+> = {
```

Comment:
I suggest a different naming for these types instead of using the word `string`

```suggestion
export interface WalletActionDetail {
  title: string;
  description: string;
  disabledDescription: string;
}

export const walletActionDetails: Record<
  WalletActionType,
  WalletActionDetail
> = {
```

Reply:
renamed here [20e79f9](https://github.com/MetaMask/metamask-mobile/pull/11330/commits/20e79f9f6747da06cc98ca81fd14d49cf471c11a)